README.md

```diff
@@ -16,9 +16,11 @@ The St. Olaf community, now in pocket size… rewritten in React Native.
 
 - [Install React Native](http://facebook.github.io/react-native/docs/getting-started.html#content)
 - [Install Yarn](https://yarnpkg.com/en/docs/install)
+- [Install Git Large File Storage](https://help.github.com/articles/installing-git-large-file-storage/)
```

Comment:
Nit: Don't expand the acronym here.

Reply:
We define every term we use. I don't see why not?

Reply:
Can we have the abbreviation here, too, then, please? "Git Large File Storage (LFS)" maybe?